tests/sentry/integrations/jira_server/test_integration.py

```diff
@@ -1042,7 +1044,8 @@ def test_migrate_plugin(self):
             key=f"{self.plugin.slug}:tid", group_id=group2.id, value="BAR-1"
         )
         org_integration = OrganizationIntegration.objects.get(integration_id=self.integration.id)
-        org_integration.config.update({"issues_ignored_fields": ["reporter", "test"]})
+        with unguarded_write(router.db_for_write(OrganizationIntegration)):
```

Comment:
A bit curious, but why does this need an unguarded write?

Reply:
It did not, so I removed. I think I was aggressive heuristically.